ansible/roles/sap-hana-ha/tasks/assertions.yml

```diff
@@ -72,3 +72,23 @@
     sap_instances:
     - '{{ sap_instance_name }}, {{ sap_hana_instance_number | int }}, 5{{ sap_hana_instance_number }}13, 5{{ sap_hana_instance_number }}14, 0.3, HDB|HDB_WORKER, GREEN'
```

Comment:
Do the `sap-hana-scaleup` and `sap-hana-scaleout` roles also need this change?

Reply:
Yes. It is required. Pushed the change